File: tutorials/csharp-tutorial/src/CSharpTutorialDemo/CSharpTutorialDemo/ConditionalStatements.cs
using System;

namespace CSharpTutorialDemo
{
    public class ConditionalStatements
    {
        public static void Example1()
        {
            bool condition = true;

            if (condition)
            {
                Console.WriteLine("The variable is set to true.");
            }
            else
            {
                Console.WriteLine("The variable is set to false.");
            }
        }

        public static void Example2()
        {
            int marks = 79;

            if (marks >= 90)
            {
                Console.WriteLine("A+");
            }
            else if (marks >= 80)
            {
                Console.WriteLine("A");
            }
            else if (marks >= 70)
            {
                Console.WriteLine("B");
            }
            else if (marks >= 60)
            {
                Console.WriteLine("C");
            }
            else if (marks >= 50)
            {
                Console.WriteLine("D");
            }
            else
            {
                Console.WriteLine("F");
            }
        }
    }
}

File: tutorials/vb-net/src/VBNetTutorialDemo/VBNetTutorialDemo/ConditionalStatements.vb
Module ConditionalStatements
    Public Sub Example1()
        Dim num1 As Integer = 7
        Dim num2 As Integer = -1

        If num1 > 0 Then
            Console.WriteLine("num1 is valid.")
        End If

        If num2 < 0 Then
            Console.WriteLine("num2 is not valid.")
        End If
    End Sub

    Public Sub Example2()
        Dim randomizer As New Random()
        Dim count As Integer = randomizer.Next(0, 5)

        Dim message As String

        If count = 0 Then
            message = "There are no items."
        Else
            message = $"There are {count} items."
        End If

        Console.WriteLine(message)
    End Sub

    Public Sub Example3()
        Dim marks As Integer = 79

        If marks >= 90 Then
            Console.WriteLine("A+")
        ElseIf marks >= 80 Then
            Console.WriteLine("A")
        ElseIf marks >= 70 Then
            Console.WriteLine("B")
        ElseIf marks >= 60 Then
            Console.WriteLine("C")
        ElseIf marks >= 50 Then
            Console.WriteLine("D")
        Else
            Console.WriteLine("F")
        End If
    End Sub
End Module
